lib/ramble/ramble/application.py

```diff
@@ -1426,9 +1449,170 @@ def format_context(context_match, context_format):
                     fom_copy['name'] = fom_name
                     context_map['foms'].append(fom_copy)
 
-                results['CONTEXTS'].append(context_map)
+                self.results['CONTEXTS'].append(context_map)
```

Comment:
It would be nice to dry out the repeated use of the magic string `'CONTEXTS'`

Reply:
Some of the other strings here are also repeated in multiple modules (e.g., 'RAMBLE_STATUS', 'RAMBLE_VARIABLES'). Ok if I create a task to come back and address all rather than just doing contexts now?

Reply:
Yep, great idea